cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -528,6 +528,57 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     clusterList.waitForPage();
   });
 
+  describe('Cluster Details Tabs', () => {
+    beforeEach( () => {
+      HomePagePo.goTo();
+      const burgerMenu = new BurgerMenuPo();
+
+      BurgerMenuPo.toggle();
+      const clusterManagementNavItem = burgerMenu.links().contains(`Cluster Management`);
+
+      clusterManagementNavItem.should('exist');
+      clusterManagementNavItem.click();
+      const clusterList = new ClusterManagerListPagePo('_');
+
+      clusterList.waitForPage();
+      clusterList.clickOnClusterName('local');
+    });
+
+    it('can navigate to Cluster Conditions Page', () => {
+      const clusterDetail = new ClusterManagerDetailImportedGenericPagePo(undefined, 'local');
```

Comment:
this can be defined before the `it()` blocks (line after `describe()`) and utilized in each test.